packages/xsnap/test/test-snapstore.js

```diff
@@ -14,11 +14,11 @@ import tmp from 'tmp';
 import { xsnap } from '../src/xsnap';
 import { makeSnapstore } from '../src/snapStore';
 
-const importModuleUrl = `file://${__filename}`;
+const importMetaUrl = `file://${__filename}`;
 
 const asset = async (...segments) =>
   fs.promises.readFile(
-    path.join(importModuleUrl.replace('file:/', ''), '..', ...segments),
+    path.join(importMetaUrl.replace('file:/', ''), '..', ...segments),
```

Comment:
I like the variable rename. I think `new URL(importMetaURL).pathname` is more correct, assuming URL is an option.

Reply:
ah... right... I must have copied this from a context where `URL` wasn't available.